fuzz-testing/src/main/scala/org/apache/comet/fuzz/Main.scala

```diff
@@ -60,13 +64,20 @@ object Main {
     val conf = new Conf(args.toIndexedSeq)
     conf.subcommand match {
       case Some(conf.generateData) =>
-        DataGen.generateRandomFiles(
-          r,
-          spark,
-          numFiles = conf.generateData.numFiles(),
-          numRows = conf.generateData.numRows(),
-          numColumns = conf.generateData.numColumns(),
+        val options = DataGenOptions(
+          allowNull = true,
+          generateArray = conf.generateData.generateArrays(),
+          generateStruct = conf.generateData.generateStructs(),
+          generateMap = conf.generateData.generateMaps(),
           generateNegativeZero = !conf.generateData.excludeNegativeZero())
+        for (i <- 0 until conf.generateData.numFiles()) {
+          ParquetGenerator.makeParquetFile(
+            r,
+            spark,
+            s"test$i.parquet",
+            numRows = conf.generateData.numRows(),
```

Comment:
`numRows` is for all rows from all generated files or for one single file? Do we have definition for it?

Reply:
This is the number of rows per file. I pushed a commit to add descriptions for all command-line options.

```
  -e, --exclude-negative-zero   Whether to exclude negative zero
  -g, --generate-arrays         Whether to generate arrays
      --generate-maps           Whether to generate maps
      --generate-structs        Whether to generate structs
  -n, --num-files  <arg>        Number of files to generate
      --num-rows  <arg>         Number of rows per file
  -h, --help                    Show help message
```